src/app/portlets/dot-edit-page/layout/components/dot-sidebar-properties/dot-sidebar-properties.component.scss

```diff
@@ -0,0 +1,7 @@
+@import 'variables';
+
+:host {
+    position: absolute;
```

Comment:
Same thing... we can't position a generic component like this... this position should be set by the parent.